web/src/components/MultiSelect.jsx

```diff
@@ -3,15 +3,16 @@ import { useRef, useState } from 'preact/hooks';
 import Menu from './Menu';
 import { ArrowDropdown } from '../icons/ArrowDropdown';
 import Heading from './Heading';
+import Button from './Button';
 
-export default function MultiSelect({ className, title, options, selection, onToggle }) {
+export default function MultiSelect({ className, title, options, selection, onToggle, onShowAll, onSolo }) {
```

Comment:
To match the other function can we cal it something like `onSelectSingle` instead of `onSolo`